doc/docs/1.10.x/reference/pipeline_spec.md

```diff
@@ -278,11 +278,18 @@ on `stdin`.
 Lines do not have to end in newline characters.
 
 `transform.env` is a key-value map of environment variables that
-Pachyderm injects into the container.
+Pachyderm injects into the container. There are environment variables that
+are automatically injected into the container. The most commonly used
+variables include:
```

Comment:
```suggestion
Pachyderm injects into the container. There are also environment variables that
are automatically injected into the container such as:
```

Reply:
make this same change in the 1.11 branch

Reply:
added.